q2_sample_classifier/assets/index.html

```diff
@@ -6,6 +6,28 @@
 
 {% block content %}
 
+{% if warn_test %}
+<div class="panel-group" id="warnings" role="tablist" aria-multiselectable="true">
+  <div class="panel panel-warning">
+    <div class="panel-heading" role="tab" id="warnings-heading">
+      <h4 class="panel-title">
+        <a role="button" data-toggle="collapse" data-parent="#warnings" href="#warnings-list" aria-expanded="true" aria-controls="warnings-list">
+          Warnings (click here to collapse/expand):
+        </a>
+      </h4>
+    </div>
+    <div id="warnings-list" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="warnings-heading">
+      <div class="alert alert-warning col-md-12">
+        <p><strong>"Displayed model performance values were calculated on the train set because estimator was 
```

Comment:
I'd say the quotes around the message are not necessary as they also are displayed in the visualisation.